### PR TITLE
brave: 1.7.92 -> 1.8.90

### PR DESCRIPTION
**Pull request by charles-dyfis-net**
###### Motivation for this change

Package update. Security fixes, bugfixes, etc.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/browsers/brave/default.nix [hunk 1]
```diff
@@ -29,6 +30,7 @@
 , libXrender
 , libXScrnSaver
 , libXtst
+, mesa_drivers
```
Comment (lourkeur) [inline pkgs/applications/networking/browsers/brave/default.nix:33]: That should make OfBorg happy without affecting the actual derivation (assuming that's required) (1/2)
```suggestion
, mesa
```

File: pkgs/applications/networking/browsers/brave/default.nix [hunk 2]
```diff
@@ -69,6 +72,7 @@ rpath = lib.makeLibraryPath [
   libXrender
   libXtst
   libuuid
+  mesa_drivers.out # libgbm.so
```
Comment (lourkeur) [inline pkgs/applications/networking/browsers/brave/default.nix:75]: 2/2
```suggestion
  mesa.drivers.out # libgbm.so
```

**Review by bbjubjub2494 — COMMENTED** [review]
Builds fine, appears to work fine (I'm a Brave user)

**Comment by lourkeur** [comment]
superseded by #87277 assuming it gets merged

**Comment by puzzlewolf** [comment]
I guess this can be closed, #87277 was merged.

**Pull request closed without merging**